Jenkinsfile

```diff
@@ -11,8 +46,38 @@ node('vetsgov-general-purpose') {
     ]]
   ]);
 
+  stage('Validate Links') {
+    dir(CONTENT_REPO) {
+      checkout scm
+    }
+
+    dir(APP_CODE_REPO) {
+      checkoutAppCode()
+      imageTag = java.net.URLDecoder.decode(env.BUILD_TAG).replaceAll("[^A-Za-z0-9\\-\\_]", "-")
+      dockerImage = docker.build("vets-website:${imageTag}")
+    }
+
+    def currentDir = pwd()
+    def dockerArgs = "-v ${currentDir}/${APP_CODE_REPO}:/application -v ${currentDir}/${CONTENT_REPO}:/${CONTENT_REPO}"
+
+    dockerImage.inside(dockerArgs) {
+      def installDependencies = "cd /application && yarn install --production=false"
+      def build = "npm --prefix /application --no-color run build -- --buildtype=vagovprod --entry static-pages 2>&1 | tee output.log"
+
+      sh installDependencies
+      sh build
+    }
+
+    def output = sh(returnStdout: true, script: "cat output.log").trim()
+    def hasError = output.indexOf('npm ERR!') > -1
+    if (hasError) {
+      commentBrokenLinks(output)
+      error(output)
+    }
+  }
+
```

Comment:
There's obviously a lot of code redundancy here with vets-website's Jenkinsfile, but I think we can address that when we implement the content deployment.